### DISCUSSION TITLE
Proposal - <podcast:warning>

### DISCUSSION THREAD

**Discussion by horrificnathan** [Podcastindex-org/podcast-namespace#454] [Tag Proposal]
A lot of media consumers utilize ratings which are further broken down by content warning for video games, movies, and TV. My suggestion is to implement a similar system in podcasts using a meta tag. This could allow apps to setup filters for their users based on content, and advertisers to choose what types of content they want their ads to appear on (theoretically this could be paired with dynamic insertion tools, where ads will automatically only load on episodes without certain content).

I've outlined suggestions for standardized ratings (more simple than other media) and content warnings (more detailed than other media) that should encompass both fiction and non-fiction podcasts:

**Everyone**

- P - Politics
- R - Religion
- f - Firearm SFX
- l - Loud SFX
- t - Traffic SFX

**Teen**

- B - Bigotry (referenced)
- D - Death (referenced)
- G - General Horror
- H - Mental/Physical Health
- I - Intoxicating Substances (referenced)
- L - Explicit Language
- S - Sex (referenced)
- T - Tense Sequences
- V - Violence (referenced)
- g - Gore SFX

**Mature**

- A - Animal Harm
- B - Bigotry (depicted)
- C - Child Harm
- I - Intoxicating Substances (depicted)
- N - Non-consent
- S - Sex (depicted)
- V - Violence (depicted)

**Comment by Marzal** [comment]
Hi, as per https://github.com/Podcastindex-org/podcast-namespace/discussions/749 and https://github.com/Podcastindex-org/podcast-namespace/discussions/695 we are closing old issues.

Please open a new [Discussions](https://github.com/Podcastindex-org/podcast-namespace/discussions) if you think that the matter it's worth pursuing.

Related: https://github.com/Podcastindex-org/podcast-namespace/issues/151
